README.md

```diff
@@ -33,6 +33,29 @@ Hint: Think about what happens in each iteration of the loop, and how often the
 loop is executed. Keep in mind that for asymptotic analysis we don't care about
 constant factors.
 
+
 Describe your reasoning and the conclusion you've come to. Your reasoning is
 most important -- you can easily find the answer, but you need to demonstrate
 that you've understood the concept. Add your answer to this markdown file.
+
+Consider the insertion sort function:
+
+```javascript
+function insertionSort(arr) {
+  for(var i = 1; i < arr.length; i++) {
+    var val = arr[i];
+    var j;
+    for(j = i; j > 0 && arr[j-1] > val; j--) {
+      arr[j] = arr[j-1];
+    }
+    arr[j] = val;
+  }
+  return arr;
+}
+```
+
+The average location of an element is $n/2$ and the function will compare it with all elements before it, approximately $(n - 1)/2$ comparisons. Thus on average, with $n$ elements, it will make $\frac{n * (n - 1)}{2}$ comparisons. Hence it will take $T(n) = \frac{n(n - 1)}{2} \in \Theta(n^2)$
+
+I certify that I have listed all sources used to complete this exercise, including the use of any Large Language Models. All of the work is my own, except where stated otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is suspected, charges may be filed against me without prior notice. 
+
+I did everything independently. But I used this 
```

Comment:
But you used what?

Reply:
I was trying to paste the link in but my computer was locking up and I had to go and forgot to fix it later. It should be fixed now.